addons/attributes/functions/fnc_openForSelection.sqf

```diff
@@ -0,0 +1,25 @@
+#include "script_component.hpp"
+/*
+ * Author: Ampersand
+ * Opens the attributes display for the selected entity.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call zen_attributes_fnc_openForSelection
+ *
+ * Public: No
+ */
+
+curatorSelected params ["_objects", "_groups", "_waypoints", "_markers"];
+
+switch (true) do {
+    case !(_groups isEqualTo []): {[_groups select 0, "Group"] call zen_attributes_fnc_open};
```

Comment:
Call `BIS_fnc_showCuratorAttributes` to open attributes - it handles not opening attributes for objects that have them disabled and opening `curatorInfoType` dialog for objects that still use that.

Reply:
Ahh, I figured out it's `_entity call BIS_fnc_showCuratorAttributes`. I was getting error with `[_entity] call BIS_fnc_showCuratorAttributes` and thought it wasn't used or something.